Github-Issue-Guide.md

```diff
@@ -0,0 +1,55 @@
+
+## Create a GitHub Account
+
+1. Visit the GitHub website at [https://github.com](https://github.com).
+
+2. Click on the "Sign up" button to create a new GitHub account.
+
+3. Follow the instructions provided by GitHub to complete the signup process.
+
+4. Verify your account by clicking on the confirmation link sent to your email.
+
+## Access Oppia's GitHub Repository
+
+1. Open your web browser and go to [Oppia's GitHub repository](https://github.com/oppia/oppia).
+
+2. You will see the main page of the Oppia repository where the development and issue tracking take place.
+
+## File a New Issue
+
+1. On the Oppia repository page, locate the "Issues" tab near the top navigation bar and click on it.
+ <img width="1437" alt="Screenshot 2023-06-08 at 23 50 59" src="https://github.com/Ash-2k3/oppia-web-developer-docs/assets/98253080/1e0e3833-d04b-4507-8251-f6039a094329">
+
+2. In the "Issues" tab, you will find a green button labeled "New issue." Click on it to create a new issue.
+<img width="1440" alt="Screenshot 2023-06-08 at 23 51 58" src="https://github.com/Ash-2k3/oppia-web-developer-docs/assets/98253080/05b279f5-24b9-4986-a7a9-352e882198f1">
+
+3. Choose type of the issue which you want to file from the list of available types.
+<img width="1440" alt="Screenshot 2023-06-08 at 23 53 15" src="https://github.com/Ash-2k3/oppia-web-developer-docs/assets/98253080/e4ac8fd7-858f-41ca-84c1-0b7a32f425d6">
+
+4. On the issue creation page, provide a concise and descriptive title for your issue in the "Title" field and follow the steps as mentioned along with each comment box.
+<img width="1440" alt="Screenshot 2023-06-09 at 00 00 18" src="https://github.com/Ash-2k3/oppia-web-developer-docs/assets/98253080/0a578025-cef5-4a6b-bb7b-dbf0ba0b8887">
+
+5. If applicable, you can include screenshots or any other supporting files by dragging and dropping them into the comment box.
+
+6. Preview your issue by clicking on the "Preview" tab and make any necessary edits.
+
+7. Once you are satisfied with your issue, click on the green "Submit new issue" button to file it.
+
+## Engage in the Issue Discussion
+
+1. After filing the issue, you will be redirected to the issue's page.
+
+2. From this point, you can monitor updates and engage in discussions related to the issue.
+
+3. If a developer requests additional information or clarification, you can provide your input in the comments section.
+
+4. Feel free to ask questions, share insights, or offer suggestions to help resolve the issue.
+
+## Conclusion
+
+Congratulations! You have successfully filed an issue on Oppia's GitHub repository. Your contribution is valuable in improving Oppia for everyone. Remember to follow notifications and actively participate in the issue's discussion to collaborate effectively with the Oppia community.
```

Comment:
The "to collaborate..." is a bit vague, maybe we can say "to help the developers fix it as soon as possible" or something like that?

Reply:
SGTM, Thanks!